docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,68 @@
+# Release 450 (7 Jun 2024)
+
+## General
+
+* Add support for the `optimizer.push-filter-into-values-max-row-count`
+  configuration property, which can define a number of rows to be pushed from a
+  filter into values. ({issue}`22105`)
+* Improve performance of the {func}`first_value` and {func}`last_value`
+  functions. ({issue}`22092`)
+* Improve performance for large clusters under heavy workloads. ({issue}`22039`)
+* Improve performance of queries with simple predicates. This optimization can
+  be disabled using the `experimental.columnar-filter-evaluation.enabled`
+  configuration property or `columnar_filter_evaluation_enabled` session
+  property. ({issue}`21375`)
+* Fix failure when loading the {doc}`/admin/event-listeners-openlineage`. ({issue}`22228`)
+
+## JDBC driver
+
+* Add support for the `assumeNullCatalogMeansCurrent` connection property. When
```

Comment:
fyi .. this is not documented on purpose since it seems to be super niche problem